stock_release_channel_cutoff/models/stock_release_channel.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2023 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+import pytz
+
+from odoo import api, fields, models
+
+
+class StockReleaseChannel(models.Model):
+    _inherit = "stock.release.channel"
+
+    cutoff_datetime = fields.Datetime()
+    cutoff_time = fields.Char(compute="_compute_cutoff_time")
+
+    @api.depends("cutoff_datetime")
+    def _compute_cutoff_time(self):
+        for rec in self:
+            if rec.cutoff_datetime:
+                # Get user's timezone
+                user_tz = pytz.timezone(self.env.user.tz or "UTC")
+                # Convert to user's timezone
+                local_time = pytz.utc.localize(rec.cutoff_datetime).astimezone(user_tz)
+                rec.cutoff_time = local_time.strftime("%H:%M")
```

Comment:
It can be simplified as below
```suggestion
                context_dt = fields.Datetime.context_timestamp(rec, rec.cutoff_datetime)
                rec.cutoff_time = context_dt.strftime("%H:%M")

```